crates/chia-sdk-driver/src/primitives/cat.rs

```diff
@@ -484,7 +484,7 @@ mod tests {
             program: tail,
             args: EverythingWithSignatureTailArgs::new(pk),
         })?;
-        let run_tail = Condition::Other(ctx.alloc(&RunTail::new(tail_program, ()))?);
+        let run_tail = Condition::Other(ctx.alloc(&RevealCatTail::new(tail_program, ()))?);
```

Comment:
RunCatTail? You're indeed revealing it, but the ultimate purpose is to run it with the given solution so the extra delta is authorized